gitpod-io/website
# Changed node types for GCP single cluster ref arch

## Description
As a result of the PR to merge the AWS single ref architecture (see https://github.com/gitpod-io/website/pull/2324), we assessed the recommended node sizes and concluded that we want to change the recommendation for GCP. Specifically: 
- **For Meta / Services nodes**, go from `n2d-standard-8` to `n2d-standard-4`. 4 cors is plenty (we use 2 in our internal preview environments, see [recommendation](https://gitpod.slack.com/archives/C01KGM9EBD4/p1658245933043589?thread_ts=1657873575.004779&cid=C01KGM9EBD4) (internal) from @meysholdt ) and is what we are recommending on the AWS side. It is further aligned with our current preflight checks, which [warn below 4 cores](https://github.com/gitpod-io/gitpod/blob/main/install/kots/manifests/kots-preflight.yaml#L201-L241). `n2d-standard-4` also has 16gb of memory, which is what we recommend and check for for services nodes. The autoscaler should be used to spin up more nodes if there is more demand. 
- **For workspace nodes** go from `n2d-standard-16` to `n2d-standard-8`: The smaller nodes save a lot of money per month, if someone ones beefier nodes (or fewer node instantiations for faster workspace start times) they can change this value. 

## Related Issue(s)

- PR where this came up: https://github.com/gitpod-io/website/pull/2324

## How to test
- Go to netlify preview, go to docs > Self-Hosted > Installation Guides > Reference Architectures > single cluster reference architecture. and ensure that all mentions of n2d-standard-x were changed and that the guide still works as before :) 

## Release Notes

```release-note
NONE
```


<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2487"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
